docs/Custom-brimcap-load-Config.md

```diff
@@ -0,0 +1,433 @@
+invoked are in the `$PATH`, the configuration can be tested outside the app to
+import a `sample.pcap` like so:
+
+```
+$ /opt/Brim/resources/app.asar.unpacked/zdeps/zed api new testpool
+$ /opt/Brim/resources/app.asar.unpacked/zdeps/brimcap load -root "$HOME/.config/Brim/data/brimcap-root" -config zeek-suricata.yml -p testpool sample.pcap
+```
+
+* **Note**: The `zdeps` directory that contains the `zed` and `brimcap`
+binaries varies per platform.
+
+|**OS Platform**|**Location**|
+|---------------|------------|
+| **Windows**   | `%USERPROFILE%\AppData\Local\Programs\Brim\resources\app.asar.unpacked\zdeps` |
+| **macOS**     | `/Applications/Brim.app/Contents/Resources/app.asar.unpacked/zdeps` |
+| **Linux**     | `/opt/Brim/resources/app.asar.unpacked/zdeps` |
+
+If successful, the new pool will appear in Brim, allowing you to browse the
+logs and open flows from the pcap via the **Packets** button.
+
+![NetFlow Pool](media/Custom-Zeek-Suricata-Pool.png)
+
+The same `brimcap load` command line can be used to incrementally add more logs
+to the same pool for additional pcaps, which was not possible pre-Brimcap. The
+setting in the Brim **Preferences** for the **Brimcap Custom YAML Path** can
+also be pointed at the path to this configuration file, which will cause it to
+be invoked when you open or drag pcap files into Brim.
+
+In examining the example Brimcap YAML, we see at the top that we've defined two
+`analyzers`.
+
+```
+analyzers:
+  - cmd: zeek-wrapper.sh
+  - cmd: suricata-wrapper.sh
+```
+
+A Brimcap analyzer has the following characteristics:
+
+1. It expects pcap input to be streamed to it via standard input (stdin)
+2. It's expected to output log files that can be further processed as soon as
+lines are appended to them (i.e. `tail` could process them)
+3. Additional per-analyzer options can be used to affect which generated logs
+are loaded and what additional processing is performed on them
+
+The analyzer invoked by Brimcap is a wrapper script as referenced in the YAML.
+In addition to taking input on stdin, it also sets Zeek to ignore checksums
+(since these are often set incorrectly on pcaps) and we also disable a couple
+of the excess log outputs. This is very similar to the Zeek Runner script that
+was included with Brim `v0.24.0`.
+
+```
+$ cat zeek-wrapper.sh 
+#!/bin/bash
+zeek -C -r - --exec "event zeek_init() { Log::disable_stream(PacketFilter::LOG); Log::disable_stream(LoadedScripts::LOG); }" local
+```
+
+We use a similar wrapper for Suricata, but here we perform some additional
+steps to work around a known Suricata bug [#4106](https://redmine.openinfosecfoundation.org/issues/4106).
+This bug causes EVE JSON events to be output that contain duplicate keys, which
+Zed is not currently equipped to tolerate ([zed/2523](https://github.com/brimdata/zed/issues/2523)). We therefore pipe the EVE JSON data through [`jq`](https://stedolan.github.io/jq/),
+which has the effect of deduplicating these references. Shell pipeline
```

Comment:
It'd be great to see brimdata/zed#2523 fixed so I don't need to include this `jq` hack, but OTOH, the hack illustrates the flexibility of Brimcap. :man\_shrugging:

Reply:
brimdata/zed#2523 has been addressed so I've taken out the `jq` hack and accompanying text. The NetFlow example shows plenty of command line hacking, so that still gets the point across for those who have the patience to read on. Marking this one as resolved.